src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -507,6 +553,16 @@ private StatusResponsePayload ConvertFrom(DurableOrchestrationStatus status)
             };
         }
 
+        private StatusResponsePayload ConvertFrom(DurableEntityStatus status)
+        {
+            return new StatusResponsePayload
+            {
+                InstanceId = status.EntityId.ToString(),
+                Input = status.State,
+                LastUpdatedTime = status.LastOperationTime.ToString("s") + "Z",
+            };

```

Comment:
I don't think `StatusResponsePayload` is the right thing to return. That class is for orchestration instances, and fields like `InstanceId`, `Input`, and `LastUpdatedTime` won't make sense to customers when they are querying for entities. Rather, we should be returning something closer to `DurableEntityStatus` which has field names that will make much more sense for entities.

One change I would make to `DurableEntityStatus`, however, is that instead of trying to serialize an `EntityId` property, we should have separate string fields for the entity key and the entity name. I expect the result should look something like this:

```json
[{
  "entityName": "cat",
  "entityId": "whiskers",
  "state": {
    "age": 2,
    "status": "hungry"
  },
  "lastOperationTime": "2019-12-11T06:30:03Z"
}]
```

A simple way to do this would be to return a hand-crafted `JObject` from this method instead of a `StatusResponsePayload`. Alternatively, you could modify `DurableEntityStatus` to serialize itself into JSON this way - which has the nice (but not required) benefit of allowing customers to easily serialize these objects in their code the same way we serialize them.